src/main/java/controller/RegisterCtrl.java

```diff
@@ -0,0 +1,114 @@
+package controller;
+
+import domain.User;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import javafx.stage.Stage;
+import service.UserService;
+import util.Controller;
+import util.UIUtil;
+
+
+/**
+ * Created by Mihai on 04.04.2017.
+ */
+public class RegisterCtrl extends Controller {
+    @FXML
+    private TextField firstNameTxt;
+
+    @FXML
+    private TextField secondNameTxt;
+
+    @FXML
+    private TextField emailTxt;
+
+    @FXML
+    private PasswordField passwordTxt;
+
+    @FXML
+    private PasswordField repeatPassTxt;
+
+    private UserService service;
+
+    @FXML
+    private void initialize() {
+        this.service = (UserService) this.get("service.user");
+    }
+
+
+    /**
+     * @return true if password is valid, false otherwise
+     * Check if password length is bigger then 5 characters
+     * Check if password field and repeat password field contain the same value
+     */
+    private Boolean validatePasswords(){
+        if (passwordTxt.getText().length() < 5){
+            UIUtil.errorAlert("Password should be at least 5 characters.");
+            return false;
+        }
+        if (!passwordTxt.getText().equals(repeatPassTxt.getText())){
+            UIUtil.errorAlert("Passwords don't match!");
+            return false;
+        }
+        return true;
+    }
+
+
+    /**
+     * @return true if textFields are not empty, false otherwise
+     * Check for each textField if it is empty
+     */
+    private Boolean notEmptyFields(){
```

Comment:
Nu trebuie sa faci asta, pentru asta exista anotarile de validari pe entitate.In exceptia aruncata (ValidationException), o sa ai toate stringurile cu erori si doar trebuie sa le afisezi.Ai putea sa si faci o functie in helper care ia un obiect ValidationException ca si parametru si afiseaza toate erorile, una pe linie, intr-un error alert.